esp-wifi/src/compat/timer_compat.rs

```diff
@@ -65,18 +68,18 @@ pub fn compat_timer_arm_us(ets_timer: *mut ets_timer, us: u32, repeat: bool) {
             timer.active = true;
             timer.periodic = repeat;
         } else {
-            debug!("timer_arm_us {:x} not found", ets_timer as usize);
+            trace!("timer_arm_us {:x} not found", ets_timer as usize);
```

Comment:
This probably should remain at a higher level

Reply:
these are all quite frequent (with BLE at least) currently while things work fine -maybe they should be kept as debug and investigated

Reply:
I _guess_ it's fine if they call `disarm` or `done` on a timer we don't know (yet) about
I'm not too sure if it's the same for `arm`? In theory if they never did `setfn` we cannot/don't need to do much - or maybe we should just initialize the timer in `arm` anyway with a no-op callback